.github/workflows/test.yaml

```diff
@@ -7,6 +7,11 @@ on:
   pull_request:
     branches: [ 'main', 'master' ]
 
+  merge_group:
+    types: [checks_requested]
+
+  workflow_dispatch:
```

Comment:
same as other pr. this is only needed if there is a desire to allow humans to trigger tests via ui.